sender/influxdb.go

```diff
@@ -48,6 +48,7 @@ type InfluxDB struct {
 	Measurement             string          `doc:"Measurement name to write to."`
 	MeasurementFromMetadata string          `doc:"Metadata key to read the measurement from. Either this or 'measurement' must be set. If both are present, 'measurement' will be used if the named metadatakey is not found."`
 	Timeout                 skogul.Duration `doc:"HTTP timeout"`
+	ConvertInts             bool            `doc:"Convert integers to influx line protocol integers"`
```

Comment:
As we've discussed, this is adding a level of technical debt - pending decisions on our end, a few thoughts:

1. The boolean should be flipped: The default behavior should be that integers get sent as integers.
2. The documentation really really REALLY should have a huge "don't do this if you don't know EXACTLY why"-warning.
3. We may want to add a skogul.Deprecated()-type function, though it's a bit premature: When this is used, we should print a warning,both because it's a generally Bad Idea in 98% of all circumstances, and because we want to remove the toggle in the future.